README.md

```diff
@@ -46,6 +46,7 @@ property-set maxConcurrentHttpConnections 100
 property-set writeTimeoutInMs 1000
 property-set readTimeoutInMs 1000
 property-set metricCacheSize 1000
+property-set externalCacheSize 1000
```

Comment:
nit: maybe rename to external**Tag**CacheSize to make it clearer what the cache is about.